css/at-rules/counter-style.json

```diff
@@ -457,19 +457,23 @@
             "description": "<code>symbols</code>",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": "91",
+                "notes": "Does not support <code>&lt;image&gt;</code> as a value for the <code>symbols</code> descriptor."
```

Comment:
It's a bit worrying when there's a repeating note. It often suggests that the data needs to change too. Since the spec says this ought to be supported, perhaps we should set `partial_implementation` as well?

Reply:
I'm not sure I understand, which data needs updating?

Reply:
Sorry for the confusion. I'm suggesting either one of:

1. Set `"partial_implementation": true` for every entry with "Does not support <code>&lt;image&gt;</code> as a value for the <code>symbols</code> descriptor".
2. Make a subfeature to capture support for `<image>` and drop the notes.

I prefer the first option, in this case.